lib/editable.dart

```diff
@@ -98,6 +98,7 @@ class Editable extends StatefulWidget {
     required this.tableHeight,
     required this.editedRowBackgroundColor,
     required this.selectedRowBackgroundColor,
+    required this.showImage,

```

Comment:
What I would do here is to pass a map of action label (string) to callback. Then add them all to the table when building.

The table should not know that there is a `showImage` callback.

Reply:
This is like this because we probably are going to change the way we options for editable.

Reply:
Ok, let's leave it like this for now, and decide later how to improve it, when we have the final design.